Sources/Kevin/InAppPayments/PaymentConfirmation/KevinPaymentConfirmationViewModel.swift

```diff
@@ -78,26 +78,6 @@ internal class KevinPaymentConfirmationViewModel : KevinViewModel<KevinPaymentCo
     }
     
     private func appendUrlParameters(urlString: String) -> URL {
```

Comment:
Just wondering, why you left a function here, while all other places you unwrapped that function? There is no benefit to have it here as well, as it just returns `FrameCustomisationHelper` 🤔

Reply:
Changed in a9290544deb2ab6e47f1348aa30a150eccc813cc